pai-management/bootstrap/prometheus/node-exporter-ds.yaml.template

```diff
@@ -56,6 +62,13 @@ spec:
           name: scrape
       - image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}gpu-exporter:{{ clusterinfo['dockerregistryinfo']['docker_tag'] }}
         imagePullPolicy: Always
+        readinessProbe:
+          exec:
+            command:
+            - python
+            - /usr/local/healthy_check.py
```

Comment:
Could you tell me when was this file (`health_check.py`) placed into this place?

Reply:
at dockerfile
https://github.com/Microsoft/pai/blob/master/pai-management/src/gpu-exporter/dockerfile